[Windows Build error] CMake build failed

Description
===========
Cloned the latest code and tried to build as per the instructions.
cmake .. -G "Visual Studio 2015" failed with below error

CMake Error at CMakeLists.txt:36 (string):
  **_string sub-command REPLACE requires at least four arguments._**


-- Could NOT find Sphinx (missing: SPHINX_EXECUTABLE)
-- Could NOT find LATEX (missing: LATEX_COMPILER)
-- Configuring incomplete, errors occurred!


Background Information / Reproduction Steps
===========================================
1) cmake v3.11.4
2) python 3.7
3) Visual Studio 2015
4) Git v2.18.0

Steps:
     1) git clone <open62541>
     2) mkdir build
     3) cd build
     4) cmake .. -G "Visual Studio 2015"


Followed the build instructions. But cmake build failed with :
**_string sub-command REPLACE requires at least four arguments._**

in CMakeLists.txt, error occurs in the below line
string(REPLACE "\n" "" OPEN62541_VER_COMMIT ${GIT_COM_ID} )

but if I add double quotes to the last argument, it works fine.
string(REPLACE "\n" "" OPEN62541_VER_COMMIT "${GIT_COM_ID}" )

Please let me know your opinion on this.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,7 +32,7 @@ set(LIB_INSTALL_DIR lib CACHE PATH "Installation path of libraries")
 if(GIT_FOUND)
     execute_process(COMMAND ${GIT_EXECUTABLE} describe --always --tags
                     RESULT_VARIABLE res_var OUTPUT_VARIABLE GIT_COM_ID WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
-    if(${res_var} EQUAL 0)
+    if(${res_var} EQUAL 0 AND NOT OPEN62541_VER_COMMIT STREQUAL "")
         string(REPLACE "\n" "" OPEN62541_VER_COMMIT ${GIT_COM_ID} )
     endif()
 endif()

